changed two images to something nicer

remyb2canada opened the pull request.

RachelMuston reviewed (commented):
Hi @remyb2canada. The new images look great! I've updated the alt text on the blog post to be a bit more descriptive for those with screen readers. If you agree, the french should also be updated. Also, I probably asked this but I'm assuming these images are all creative commons? No copyright?

remyb2canada commented:
> Hi @remyb2canada. The new images look great! I've updated the alt text on the blog post to be a bit more descriptive for those with screen readers. If you agree, the french should also be updated. Also, I probably asked this but I'm assuming these images are all creative commons? No copyright?

Some are copyrighted, I've added mention of the authors in the image. The others are creative commons (I still mention in the blog's notes section where I got them).
